src/modules/commander/commander_params.c

```diff
@@ -87,58 +87,86 @@ PARAM_DEFINE_FLOAT(TRIM_PITCH, 0.0f);
 PARAM_DEFINE_FLOAT(TRIM_YAW, 0.0f);
 
 /**
- * Empty cell voltage.
+ * Number of battery cells.
  *
- * Defines the voltage where a single cell of the battery is considered empty.
+ * Defines the number of cells the attached battery consists of.
  *
  * @group Battery Calibration
- * @unit V
+ * @unit S
+ * @min 1
+ * @max 10
  */
-PARAM_DEFINE_FLOAT(BAT_V_EMPTY, 3.4f);
+PARAM_DEFINE_INT32(BAT_N_CELLS, 1);
 
 /**
- * Full cell voltage.
+ * Full battery cell voltage.
  *
- * Defines the voltage where a single cell of the battery is considered full.
+ * Defines the voltage at which one battery cell is considered full.
  *
  * @group Battery Calibration
  * @unit V
+ * @min 0.0
  */
-PARAM_DEFINE_FLOAT(BAT_V_CHARGED, 4.2f);
+PARAM_DEFINE_FLOAT(BAT_V_FULL, 0.0f);
```

Comment:
That's got to be the 90% case though? Need feedback on that. You could add a comment in the long description that the default is for lipos. If not, lipo you'll need to adjust.